KarhooUISDK/Translations/Base.lproj/Localizable.strings

```diff
@@ -29,7 +30,7 @@
 "Text.Generic.Surname" = "Surname";
 "Text.Generic.Email" = "Email address";
 "Text.Generic.PhoneNumber" = "Phone number";
-"Text.Generic.Comment" = "Comment";
+"Text.Generic.Comment" = "Comment for the driver";
```

Comment:
Missing `(optional)` in the text I think

Reply:
I took the texts from the story and the "optional" text is translated correctly, from what I checked, not sure where to use it exactly (at least not inside this story)